llpc/test/shaderdb/core/TestEnableImplicitInvariantExports.vert

```diff
@@ -24,7 +24,7 @@ void main()
 ; WITHOUT_IIE: %[[val:.*]] = extractvalue [4 x <4 x float>] %{{.*}}, 3
 ; WITHOUT_IIE: %[[mul:.*]] = fmul <4 x float> %[[val]], %{{.*}}
 ; WITHOUT_IIE: %[[arg:.*]] = fadd <4 x float> %{{.*}}, %[[mul]]
-; WITHOUT_IIE-NEXT: call void @lgc.output.export.builtin.Position.i32.v4f32(i32 0, <4 x float> %[[arg]]) #0
```

Comment:
What was that attribute for?

Reply:
This attribute is just {nounwind} and it is hard-coded to 0. With my change, the attribute is mapped to 1 so I remove it and believe it is meaningless to check this pattern.